osf/migrations/0125_add_datacite_file_metadata_schema.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-08-16 19:24
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from django.db import migrations
+from osf.models import FileMetadataSchema
+
+logger = logging.getLogger(__file__)
+
+def add_datacite_schema(*args):
+    with open('osf/utils/metadata/schemas/datacite.json') as f:
+        jsonschema = json.load(f)
+    FileMetadataSchema.objects.create(
```

Comment:
Not sure if this would ever happen, but is there a chance when this migration is run this `FileMetadataSchema` exists already and so this `create` would fail? Maybe a `get_or_create` just for an extra measure of saftey!